### ISSUE TITLE
how to handle AutoRedirect?

### ISSUE THREAD

**Issue by cd0304** [mitre/HTTP-Proxy-Servlet#52]
my client send url to my server , throw this proxy , and my server want to redirect other web site , but this redirect seem to be auto handel by the proxy ,my client did not receive the 302 or 303 redirect message .i do not want this 

how do do that , let proxy do not autoRedirect , let browser client handle it by it self?

**Comment by dsmiley** [comment]
Yes -- I believe the redirects _may_ be handled by HttpComponents; try looking for some flag or another to tell it to not do it.
